plugins/Wallet/js/transactions.js

```diff
@@ -90,7 +90,7 @@ function makeTransaction(txn) {
 	element.find('.time').html(time);
 
 	// Set transaction positive or negative
-	if (txn.value.isNegative()) {
+	if (txn.value < 0) {
```

Comment:
It works. For some reason I don't understand, when you call toPrecision() on the BigNum it seems to be cast to a real number and isNegative() is no longer an available method. I tested it on negative and positive cases.